src/inputCL.cpp

```diff
@@ -180,7 +180,20 @@ void InputCL::read_args( int argc, char *argv[] )
                 neighbor_type = NEIGH_VERLET_2D;
             if ( ( strcmp( argv[i + 1], "VERLET_CSR" ) == 0 ) )
                 neighbor_type = NEIGH_VERLET_CSR;
+            if ( ( strcmp( argv[i + 1], "TREE" ) == 0 ) )
+                neighbor_type = NEIGH_TREE;
             ++i;
+#ifndef CabanaMD_ENABLE_ARBORX
+            if ( neighbor_type == NEIGH_TREE )
+            {
+                if ( do_print )
+                {
+                    std::cout << "ArborX requested, but not compiled!"
+                              << std::endl;
+                }
```

Comment:
I feel like you should print the error message unconditionally in that case.  Also you might want to put it to `std::cerr`.

Reply:
This is only to restrict printing to one rank, but I agree on printing the error